Tests/TestingApps/PurchaseTesterSwiftUI/Shared/ReceiptInspector.swift

```diff
@@ -73,8 +73,12 @@ struct ReceiptInspectorView: View {
     func inspectReceipt() async {
         do {
             guard !encodedReceipt.isEmpty else { return }
-            parsedReceipt = try PurchasesReceiptParser.default.parse(base64String: encodedReceipt).debugDescription
-            verifyReceiptResult = await ReceiptVerifier().verifyReceipt(base64Encoded: encodedReceipt,
+            // in kibana, receipts get encoded with extra `\`s
+            let receiptWithoutForwardSlashes = encodedReceipt.replacingOccurrences(of: "\\", with: "")
+            // just in case you accidentally copied with extra double-quotations
+            let receiptWithoutQuotations = receiptWithoutForwardSlashes.replacingOccurrences(of: "\"", with: "")
```

Comment:
FFTI, should we only remove start/end characters instead of all occurences? I think this should also work though, but in order to minify possible changes.

Reply:
I thought about it, but I don't think we stand to gain much? 

Like, the receipt won't work locally unless it's a valid receipt (save for the double-quotes), and it won't validate against `/verifyReceipt` unless the contents are unaltered, so it's pretty safe to just assume you can remove them all in the easiest way possible

Reply:
Yeah... I guess if it ever could be a valid character it would be a problem, but I don't see that happening. It's fine as it is 👍